README.md

```diff
@@ -3,21 +3,64 @@ README Languages:  [DE](README/README-DE.md) | [ES](README/README-ES.md) | [FR](
 [PT](README/README-PT.md) | [RU](README/README-RU.md) |
 [CN](README/README-CN.md) | [TW](README/README-TW.md)
 
-The Real-time File Processing reference architecture is a general-purpose, event-driven, parallel data processing architecture that uses [AWS Lambda](https://aws.amazon.com/lambda). This architecture is ideal for workloads that need more than one data derivative of an object. This simple architecture is described in this [diagram](https://s3.amazonaws.com/awslambda-reference-architectures/file-processing/lambda-refarch-fileprocessing.pdf) and ["Fanout S3 Event Notifications to Multiple Endpoints" blog post](https://aws.amazon.com/blogs/compute/fanout-s3-event-notifications-to-multiple-endpoints/) on the AWS Compute Blog. This sample application demonstrates a Markdown conversion application where Lambda is used to convert Markdown files to HTML and plain text.
+The Real-time File Processing reference architecture is a general-purpose, event-driven, parallel data processing architecture that uses [AWS Lambda](https://aws.amazon.com/lambda). This architecture is ideal for workloads that need more than one data derivative of an object. 
+
+In this example application we deliver the notes from an interview in markdown format into S3 and we utilise CloudWatch Events to trigger multiple processing flows.
 
 ## Architectural Diagram
 
 ![Reference Architecture - Real-time File Processing](img/lambda-refarch-fileprocessing-simple.png)
 
+## Application Components
+
+### Event Trigger
+Unlike batch processing, in this architecture we process each individual file as it arrives. To achive this we utilise [CloudWatch Events](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/WhatIsCloudWatchEvents.html) and [CloudTrail](https://aws.amazon.com/cloudtrail/). We write a CloudWatch Events rule which checks for S3 PutObject API calls into our Source Bucket from CloudTrail. Everytime the PutObject API is called this creates a CloudTrail log which our rule translates into an event represented as a JSON object. In our rule we also define targets which our JSON event object is delivered to, which in this scenario is 4 seperate [SQS Queues](https://aws.amazon.com/sqs/) for 4 different worflows. Other target types include AWS Lambda Functions, Kinesis Data Streams, Simple Notification Service, Step Functions state machines, ECS tasks and more.
```

Comment:
A CloudWatch Events rule detects a S3 PutObject API to our Source Bucket from CloudTrail.  Our rule defines targets that receive our JSON event object.  In our example, four seperate [SQS Queues](https://aws.amazon.com/sqs/) receive the events. Other target types include AWS Lambda Functions, Kinesis Data Streams, Simple Notification Service, Step Functions state machines, ECS tasks and more.  Please refer to [What is Amazon CloudWatch Events?](https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/WhatIsCloudWatchEvents.html) for more information about eligible targets.

Reply:
I did think about doing this. It's a duplicate of the top CloudWatch link however I think it makes sense. I'll update it.